sonar-text-dotnet/pipeline_scripts/build_and_pack_solution.ps1

```diff
@@ -0,0 +1,43 @@
+function CheckIfSuccessful($StepName) {
+if(!$?) {
+    Write-Host "sonar-text-dotnet ${StepName} failed."
+    Exit $LASTEXITCODE
+}
+}
+
+Set-Location $env:PROJECT_DIR
+$SIGN_DOTNET_ASSEMBLY = "$CIRRUS_BRANCH" -eq "master" -or "$CIRRUS_BRANCH".startsWith("branch-.*")
```

Comment:
It is the same logic as in the previous pipeline.

Reply:
shouldn't be `"$CIRRUS_BRANCH".startsWith("branch-.")` instead? 

I think that the startsWith will read the `*` as it is and not as a pattern.

Reply:
The . is also not needed. Fix is on its way.